BridgeCareApp/BridgeCare/EntityClasses/PendingCrudSupport.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace BridgeCare.EntityClasses
+{
+    public class PendingCrudSupport
```

Comment:
Is it just defined for taking care of it later?

Reply:
will check it is used and will delete if not